clang/unittests/Tooling/SourceCodeTest.cpp

```diff
@@ -374,11 +374,10 @@ TEST(SourceCodeTest, getAssociatedRangeWithComments) {
 
   // Does not include comments when only the decl or the comment come from a
   // macro.
```

Comment:
Comment isn't true any more. The comment on line 370 is also confusing to me, since... it isn't including the comment? Unless I'm missing something there.

Reply:
Good spot, can't believe I removed the FIXME and not the comment explaining the FIXME 😵‍💫